modules/Dash/resources/dash.js

```diff
@@ -139,13 +139,32 @@
 			return drmConfig;
 		},
 
+		getShakaConfig: function(){
+            var config = this.getConfig("shakaConfig");
+            if (this.getPlayer().plugins &&this.getPlayer().plugins.closedCaptions) {
+                try {
+                    var closedCaptions = this.getPlayer().plugins.closedCaptions;
+                    var textLang = closedCaptions.getUserLanguageKeyPrefrence();
+                    if (!textLang) {
+                        textLang = closedCaptions.getConfig('defaultLanguageKey');
+                    }
```

Comment:
better to code 
`var textLang = closedCaptions.getUserLanguageKeyPrefrence() || closedCaptions.getConfig('defaultLanguageKey');`